kernel_gateway/tests/notebook_http/test_request_utils.py

```diff
@@ -142,3 +142,13 @@ def test_format_request_code_escaped(self):
         #Get the value of REQUEST = "{ to test for equality
         test_request_js_value = request_code[request_code.index("\"{"):]
         self.assertEqual(test_request, json.loads(test_request_js_value), "Escaped Request code was not formatted correctly")
+
+    def test_format_request_without_a_kernel_language_arg(self):
+        test_request = ('''{"body": "", "headers": {}, "args": {}, "path": {}}''')
+        request_code = format_request(test_request)
+        self.assertTrue(request_code.startswith("REQUEST"), "Call format_request without a kernel_language argument was not formatted correctly")
+
+    def test_format_request_with_a_kernel_language_arg(self):
+        test_request = ('''{"body": "", "headers": {}, "args": {}, "path": {}}''')
+        request_code = format_request(test_request, 'perl')
+        self.assertTrue(request_code.startswith("my $REQUEST"), "Call format_request with a kernel language argument was not formatted correctly")
```

Comment:
Thanks. Would you mind adding two more of these tests, with languages 'python' and 'scala'? Python is kind of a default language, and I've seen special treatment of Scala for the comment_prefix.

These unit tests cover the changes in format_request. I would feel safer if we also had unit tests that cover the creation of the handler, where you added the new argument.

Reply:
I'll add the more languages unit tests to format_request. 

But for the creation of the handler I didn't find any existed tests for it ... maybe we could add them later?